install/install.sh

```diff
@@ -242,6 +242,33 @@ check_bin_dir() {
   fi
 }
 
+is_build_available() {
+  local arch="$1"
+  local platform="$2"
+
+  local target="${arch}-${platform}"
+  local good
+  
+  good=$(
+    for t in x86_64-unknown-linux-gnu x86_64-unknown-linux-musl i686-unknown-linux-musl x86_64-apple-darwin x86_64-pc-windows-msvc; do
```

Comment:
Could we move this to some array (or pseudo-array) so that there's a single obvious source of truth for which platforms are supported by the install script?

If that's too much of a pain in POSIX I'm not too attached to this request.

Reply:
Done!